Gor_Simonyan/python/25_05_2021/Paskal.py

```diff
@@ -0,0 +1,29 @@
+G_list = []
+def Paskal(n):
+    count = 0
+    for i in range(0,n):
+        S_list = []
+        if count == 0:
+            S_list.append(1)
+            G_list.append(S_list)
+        else:
+            for j in range(0,count+1):
+                if j-1 < 0:
+                    x = G_list[i-1][j]
+                elif j == count:
+                    x = G_list[i-1][j-1]
+                else:
+                    x = G_list[i-1][j-1] + G_list[i-1][j]
+                S_list.append(x)
+            G_list.append(S_list)
+        count+=1
+    for i in G_list:
+        print(i,'\n')
+while True:
+    try:
+        n = int(input("Write number of row! - "))
+    except(ValueError):
+        print("plz, Write number!")
+        continue
+    break
+Paskal(n)
```

Comment:
1. ![image](https://user-images.githubusercontent.com/80816144/119561464-44bf0900-bdb6-11eb-99b4-c7d26c998476.png)
- value validation is missing

2. add limitations for input(max value)  -> to be user-friendly
![image](https://user-images.githubusercontent.com/80816144/119561622-7932c500-bdb6-11eb-9dbd-fb91ae1b5a02.png)